docs/build/p2p-node/index.md

```diff
@@ -276,111 +205,326 @@ The default configuration with all available options is accessible [here](https:
+#### Open ports for incoming P2P connections
 
-  To receive an event when a new document has been shared with your node, add your WebHook endpoint in the config.yaml file:
+To accept the incoming P2P connections, you will need to open two ports for incoming TCP connections.
 
-  ```yaml
-  notifications:
-    endpoint: "http://localhost:8080/endpoint/"
-  ```
+- P2P Port: open ingress/egress. This port will be configured under `p2p` `port` in your config.
+- API Port: restrict at will, only you or your upstream systems should need to talk to it. This port will be configured as `nodeport` in your config.
 
-  For more information, see the [Notification Payload](https://centrifuge-os-node-api-4.api-docs.io/0.0.6/models/notification-message).
+### Running the Centrifuge POD after creating the configuration
 
-- Ethereum max gas price
+You can run the Centrifuge POD using the `config.yaml` file you created:
 
-  To adjust the maximum gas price (in wei) you are willing to pay per transaction:
+```bash
+$ centrifuge run -c /<PATH-TO-CONFIG-DIR>/config.yaml
+```
 
-  ```yaml
-  ethereum:
-    maxGasPrice: "xx000000000"
-  ```
+Replace the `PATH-TO-CONFIG-DIR` with the location of the `config.yaml` file.
 
-  Note: 20 Gwei would be "20000000000".
+### Post Install Verification
 
-- Disable Pre-commit
+To make sure that your Centrifuge POD setup was successful and is running properly you can ping your node.
 
-  The pre-commit option configures the node to acquire a lock on the anchor that should be anchored next before asking other peers to sign the message. This setting is enabled by default. We do not recommend disabling it. Only do so if you know what you are doing.
+```bash
+$ curl -X GET "http://localhost:8082/ping" -H "accept: application/json"
+```
 
-  ```yaml
-  anchoring:
-    precommit: false
-  ```
+It will return (e.g. Catalyst):
 
-### Open ports for incoming P2P connections
+`{"version":"...","network":"catalyst"}`
 
-To accept the incoming P2P connections, you will need to open two ports for incoming TCP connections.
+---
 
-- P2P Port: open ingress/egress. This port will be configured under `p2p` `port` in your config.
-- API Port: restrict at will, only you or your upstream systems should need to talk to it. This port will be configured as `nodeport` in your config.
+## Accounts
 
-## Post Install Verification
+[//]: # (TODO: Update the swagger link to the latest version.)
+The `Accounts` section of our [swagger API docs](https://app.swaggerhub.com/apis/centrifuge.io/cent-node/2.1.0#/Accounts) provides
+an overview of all the endpoints available for handling accounts.
 
-To make sure that your Centrifuge node setup was successful and is running properly you can ping your node.
+---
 
-```bash
-$ curl -X GET "http://localhost:8082/ping" -H "accept: application/json"
+An account is the POD representation of the user that is performing various operations. This identity of this account
```

Comment:
Not sure I get this. 

This identity of this account, or this identity of this user?

Reply:
Typo on my part, this should have been `The identity...`, in which case it makes sense to me since the account has a field named `identity`. WDYT?

Reply:
Yes, definitely!